aten/src/ATen/core/Reduction.h

```diff
@@ -1,5 +1,4 @@
-#ifndef REDUCE_H
-#define REDUCE_H
+#pragma once
 
 namespace Reduction {
```

Comment:
Not your fault... but what a weird namespace...

Reply:
Just curious, what's the appropriate way to do this? Put it in detail? I wasn't able to use a scoped enum here, but I still wanted them together in one namespace.

Reply:
Could you not used a scoped enum because it is C++ or is there another reason?